dns/bind/pkg-descr

```diff
@@ -10,8 +10,8 @@ Plugin Changelog
 
 1.4
 
-* Add Bing Strict Search
-* Add 4 new blacklists to DNSBL
+* Add Bing and DuckDuckGo Strict Search
+* Add 5 new blacklists to DNSBL (incl. StevenBlack List)
```

Comment:
Usually better not to name numbers because they can always change as happened here. From the user perspective only the blacklist names matter so either unspecified batch or naming all of them. ;)